config/openshift/v4_12_exp/translate.go

```diff
@@ -295,3 +297,28 @@ func validateMCOSupport(mc result.MachineConfig, ts translate.TranslationSet) re
 	}
 	return cutil.TranslateReportPaths(r, ts)
 }
+
+// fcos v1.5.0-exp generates a user.cfg file using append; however, OpenShift config
+// does not support append (since MCO does not support it). Let change the file to use contents
+func translateUserGrubCfg(config *types.Config, ts *translate.TranslationSet) translate.TranslationSet {
+	newMappings := translate.NewTranslationSet("json", "json")
+	for i, file := range config.Storage.Files {
+		if file.Path == "/boot/grub2/user.cfg" {
+			if len(file.Append) == 1 {
```

Comment:
The sad path should error out early so the happy path can avoid the extra indent.

Reply:
Sorry where are we avoiding the extra indent?

Reply:
Starting on the next line.